mlflow/server/__init__.py

```diff
@@ -22,6 +23,11 @@
 for http_path, handler, methods in handlers.get_endpoints():
     app.add_url_rule(http_path, handler.__name__, handler, methods=methods)
 
+if os.getenv(PROMETHEUS_EXPORTER_ENV_VAR):
+    from mlflow.server.prometheus_exporter import activate_prometheus_exporter
+    os.makedirs(os.getenv(PROMETHEUS_EXPORTER_ENV_VAR), exist_ok=True)
```

Comment:
@t-henri Looks like the `exist_ok` flag is unsupported in Python 2. Can we do something like this instead?:
```
prometheus_metrics_path = os.getenv(PROMETHEUS_EXPORTER_ENV_VAR)
if not os.path.exists(prometheus_metrics_path):
    os.makedirs(prometheus_metrics_path)
```

Reply:
Ha yes sorry my bad I forgot the python 2 support, I just updated the PR.